For_Andrey/main1.1.py

```diff
@@ -0,0 +1,163 @@
+﻿import random  # from random import randint (вызов - randint(2,5))
+import math
+import operator  # сортировка
+
+
+class Car:
+    """Use as template for creating cars"""
+    cost_car = 10000
+
+    def __init__(self, type_enj: str, tank_val: int):
+        assert tank_val in (60, 75), 'Incorrect value fuels tank (tank_val)'  # проверка
+        self.kmage = random.randint(55000, 286000)
+        self.tank_val = tank_val
+        self.type_enj = type_enj
+        if type_enj == 'diesel':
+            self.amort = 105
+            self.cost_rep = 700
+            self.fuel_rate = 6 / 100
+            self.kmage_rep = 150000
+            self.fuel_cost = 1.8
+        elif type_enj == 'petrol':
+            self.amort = 95
+            self.cost_rep = 500
+            self.fuel_rate = 8 / 100
+            self.kmage_rep = 100000
+            self.fuel_cost = 2.4
+        else:  # лучше через assert
+            x = 5 / 0
+            print('Incorrect type engine (type_enj)')

```

Comment:
print('Incorrect type engine (type_enj)') - would not work!
f'Incorrect type engine (type_enj)'